cmd/waku/server/rpc/relay.go

```diff
@@ -164,11 +164,7 @@ func (r *RelayService) PostV1AutoMessage(req *http.Request, args *RelayAutoMessa
 		r.log.Error("publishing message", zap.Error(err))
 		return err
 	}
-	if msg.ContentTopic == "" {
```

Comment:
Is it possible to publish messages without content topic now? or is this condition captured somewhere else? content topic in https://rfc.vac.dev/spec/14/ is a `MUST` altho I'm not sure if it will still allow empty.

Reply:
No, i removed this check as it is duplicate. 
This is already handled inside the relay.Publish method which would fail if contentTopic is empty.

Since this is an autosharding API where pubsubTopic is not specified, contentTopic becomes mandatory.